src/pages-and-resources/discussions/app-list/AppCard.test.jsx

```diff
@@ -1,17 +1,71 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, queryByLabelText } from '@testing-library/react';
+
+import AppCard from './AppCard';
+import messages from './messages';
+
 describe('AppCard', () => {
+  let app;
```

Comment:
Do we need these Global variables? Would it work if the variables are declared in the beforeEach method?

Reply:
No it'll not work if we declares variable in beforeEach. They are not access out side of their scope.